### ISSUE TITLE
Loading the saved model

### ISSUE THREAD

**Issue by Pranav-India** [jvishnuvardhan/Stackoverflow_Questions#1]
Hi @jvishnuvardhan 
I have saved the tensorflow model using model.save() command and now I want to load it and resume the training, when I reload the model there is a warning which says 
`WARNING:tensorflow:No training configuration found in save file, so the model was *not* compiled. Compile it manually.
`

**Comment by jvishnuvardhan** [comment]
@Pranav-India Looks like you have saved a model without a training configuration (without model.compile). If you didn't compile earlier, then you need to compile (to tell what optimizer and loss function to use during training) before training the model.

Generally, we define the model architecture, compile it and train it for few epochs and save the model. Later, reload the model and continue training where it was left last time. For further questions, Share any standalone code if you have.  Thanks!

**Comment by Pranav-India** [comment]
Hi @jvishnuvardhan  
This is the structure of the code if you need more details of the code please let me know I will provide more details.

model
`self.actor_model = actor(self.state_dim , self.action_dim , actor_lr )`

definition of class actor

    class actor():

    def __init__(self, state_input_shape ,action_dim ,actor_lr ):
        self.state_input_shape = state_input_shape
        self.action_dim = action_dim
        self.std_bound = [1e-2, 1.0]
        self.model = self.create_model()
        self.opt = tf.keras.optimizers.Adam(actor_lr)

    def create_model(self,):
        input_ = Input((32,32,2))

        conv_1 = Conv2D(64, (3, 3),  activation='relu',input_shape=(32, 32, 2))(input_)
        max_p_1 = MaxPooling2D((2, 2))(conv_1)
        drop_1 = Dropout(0.5)(max_p_1)
        conv_2 = Conv2D(16, (3, 3), activation='relu')(drop_1)
        max_p_2 = MaxPooling2D((2, 2))(conv_2)
        drop_2 = Dropout(0.5)(max_p_2)
        conv_3 = Conv2D(16, (3, 3),  activation='relu')(drop_2)
        den_1 = Flatten()(conv_3)
        den_2 = Dense(32,  activation='relu')(den_1)
        drop_3 = Dropout(0.5)(den_2)
        den_3 = Dense(32)(drop_3)

        mu = Dense(self.action_dim , activation = 'softmax')(den_3)
        #mu_out = Lambda(lambda x:x)(mu)
        #std = Dense(self.action_dim , activation = 'softmax')(den_3)

        return tf.keras.models.Model(input_ ,mu)


back-propagation 

     grads_a = tape.gradient(total_loss, self.actor_model.model.trainable_variables)
     self.actor_model.opt.apply_gradients(zip(grads_a, self.actor_model.model.trainable_variables))
